runtime/eventlog.c

```diff
@@ -306,49 +361,35 @@ void caml_ev_counter(ev_gc_counter counter, uint64_t val)
   post_event(0, counter, 0, val, EV_COUNTER);
 }
 
-static uint64_t alloc_buckets [20] = {0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0};
-
 /* This function records allocations in caml_alloc_shr_aux in given bucket sizes
    These buckets are meant to be flushed explicitly by the caller through the
    caml_ev_alloc_flush function. Until then the buckets are just updated until
    flushed.
+   TODO(engil): we actually need to record these in Multicore.
+   Since the allocator is much different in Multicore it is unclear where and if
+   we want these.
```

Comment:
These counters give information about the allocation patterns of the OCaml program, not about the allocator itself. So yes, we do want them in Multicore, eventually.